AnalyticsData/.OwlBot.yaml

```
@@ -0,0 +1,3 @@
+deep-copy-regex:
+    - source: /google/analytics/data/v1beta/.*-php/(.*)
+      dest: /owl-bot-staging/AnalyticsData/v1beta/$1
```

Are we intentionally no longer updating `v1alpha`?

Also, do we want to use wildcards on the versions here for consistency?
```suggestion
    - source: /google/analytics/data/(v1beta)/.*-php/(.*)
      dest: /owl-bot-staging/AnalyticsData/$1/$2
```